### ISSUE TITLE
Add 'existing paired alignment' to complex pipeline

### ISSUE THREAD

**Issue by nrollins-Ag** [debbiemarkslab/EVcouplings#247]
For some applications, such as protein-RNA complexes, I'd like to pair sequences by a custom method- and input already paired seqs (2 alignments where sequence X in ali 1 is paired with sequence X in ali 2) into pipeline.

This means skipping the 'genome distance' or 'best hit' step in the concatenation pipeline

**Comment by thomashopf** [comment]
Yep definitely a good idea - pinging @aggreen re protein-protein pipeline

For protein-RNA, besides the obvious hack of putting the run through the protein complex pipeline, we will need to think this through in more detail, and maybe finally create the appropriate protocols and pipelines...

**Comment by aggreen** [comment]
Yeah, the hack version of doing this now is if you use the best hit pipeline, use 'input existing alignment' for both of the monomer stages, and provide an input annotation file that has a unique identifier for each pair of sequences that you want to pair. Then best hit will just pair up all the sequences with the same annotation. But I agree it deserves a formal solution
